### DISCUSSION TITLE
Winlogbeat to Pfelk

### DISCUSSION THREAD

**Discussion by tbaror** [pfelk/pfelk#344] [Q&A]
Hello,

I am trying to send logs from few windows server using Winlogbeat , configured the to send logs to the correct Beats port,
 but no index created the no logstash error, also diabled firewall on both .
any idea ?
Thanks

**Comment by a3ilson** [comment]
Sorry that's beyond the scope of this project but I would recommend looking at the logs for both elastic and logstash to see if there are any issues.
